trlx/trainer/accelerate_base_trainer.py

```diff
@@ -487,7 +487,9 @@ def learn(self):  # noqa: C901
                     self.iter_count += 1
 
                     if self.iter_count % self.config.train.checkpoint_interval == 0:
-                        self.save()
+                        subfolder = f"checkpoint_{self.iter_count}"
```

Comment:
can you zero left pad the iter count? will make the folders be visually sorted :)